src/open_inwoner/templates/pages/search.html

```diff
@@ -13,35 +13,10 @@ <h1 class="utrecht-heading-1">{% trans "Zoekresultaten voor " %} "{% firstof sea
     {#  facets and results section #}
     <div class="grid">
     {% if paginator.count %}
+        {#  start search filters #}
         {% if search_filter_categories or search_filter_tags or search_filter_organizations %}
             <aside class="grid__sidebar grid__filters" aria-label="{% trans "Zoekfilters" %}">
-                {% if search_filter_categories and search_form.categories.field.choices|length != 0 %}
-                    {% define 0 as open_index %}
-                {% elif search_filter_tags and search_form.tags.field.choices|length != 0 %}
-                    {% define 1 as open_index %}
-                {% elif search_filter_organizations and search_form.organizations.field.choices|length != 0 %}
-                    {% define 2 as open_index %}
-                {% else %}
-                    {% define 3 as open_index %}
-                {% endif %}
-
-                {% if open_index <= 2 %}
-                <div class='filter__reset'>
-                    {% button transparent=True type='button' inline=True text=_('Wis alle?') %}
-                </div>
-                {% endif %}
-
-
-                {% if search_filter_categories and search_form.categories.field.choices|length != 0 %}
-                    {% include "components/Filter/Filter.html" with field=search_form.categories form_id="search-form" initial_open=open_index|is_same:0 only  %}
-                {% endif %}
-                {% if search_filter_tags and search_form.tags.field.choices|length != 0  %}
-                    TAGS
-                    {% include "components/Filter/Filter.html" with field=search_form.tags form_id="search-form" initial_open=open_index|is_same:1 only %}
-                {% endif %}
-                {% if search_filter_organizations and search_form.organizations.field.choices|length != 0 %}
-                    {% include "components/Filter/Filter.html" with field=search_form.organizations form_id="search-form" initial_open=open_index|is_same:2 only %}
-                {% endif %}
+                {% search_filters search_form form_id="search-form" search_filter_categories search_filter_tags search_filter_organizations %}
```

Comment:
Keyword arguments (`form_id="search-form"`) must come after positional arguments (`search_filter_categories` etc.). This causes a `TemplateSyntaxError` which in turn causes `expect(page.locator(".search-results__item")).to_have_count(2)` to fail.